romdisk/sleep.asm

```diff
@@ -0,0 +1,94 @@
+; SPDX-FileCopyrightText: 2023 Shawn Sijnstra <shawn@sijnstra.com>
+;
+; SPDX-License-Identifier: Apache-2.0
+
+        INCLUDE "zos_sys.asm"
+
+        SECTION TEXT
+
+        EXTERN error_print
+
+        MACRO ERR_CHECK goto_label
+                or a
+                jp nz, goto_label
+        ENDM
+
+        ; "sleep" command main function
+        ; Parameters:
+        ;       HL - ARGV
+        ;       BC - ARGC
+        ; Returns:
+        ;       A - 0 on success
+        PUBLIC sleep_main
+sleep_main:
+        ; Check that argc is 2 (command itself is part of argc)
+        ld a, c
+        cp 2
+        jp nz, _sleep_usage
+        ; Retrieve the filename given as a parameter
+        inc hl
+        inc hl  ; skip the first pointer
+        ld c, (hl)
+        inc hl
+        ld b, (hl)
+        ld      a,(bc)
```

Comment:
`romdisk`'s `strutils.asm` files already provides `parse_int` routine, it would be better to re-use it, moreover, it support both hexadecimal and decimal notations

Reply:
I didn't see that before - excellent! Converted and updated.